client/src/components/QA/QaIndex.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const QaIndex = (props) => {
+  return (
+    <div>
+      <h1>This is the QA section. deal with it</h1>
+      <h3>wow. so much qa here. much amazing</h3>
```

Comment:
Change this H3 text at the end to 'much wow'

Reply:
done